Client/Ecosia/Migration/EcosiaHistory.swift

```diff
@@ -112,4 +132,8 @@ final class EcosiaHistory {
         }
         return .init(domains: domains, sites: Array(mappedSites.values), visits: visits)
     }
+
+    static func cap(_ items: [(Date, Core.Page)], for days: Int) -> [(Date, Core.Page)] {
+        return items.filter({ $0.0.timeIntervalSinceNow > Double(-days * 24 * 60 * 60) })
```

Comment:
```suggestion
        items.filter {
                Calendar.current.dateComponents([.day], from: $0.0, to: .init()).day! < days
            }
```

Reply:
definetely a more appealing API 👍  I'll check it out and see how it performs